src/sections/pools/stablepool/details/PoolValue.tsx

```diff
@@ -1,59 +1,78 @@
-  total,
-  className,
-  totalDisplay,
-}: PoolValueProps) => {
+export const PoolValue = ({ pool, className }: PoolValueProps) => {
   const { t } = useTranslation()
 
+  const { data, isLoading } = useVolume(pool.id.toString())
+
+  const { total, totalDisplay } = pool
+
+  const percentInOmnipool = totalDisplay
+    .dividedBy(total.value)
+    .multipliedBy(100)
+
   return (
     <div sx={{ flex: "column", justify: "end" }} className={className}>
       <div sx={{ flex: "row", justify: "space-between" }}>
-        <div sx={{ flex: "column", gap: 10, width: ["auto", 118] }}>
-          <div sx={{ flex: "row", align: "center", gap: 6 }}>
-            <Text fs={13} color="basic400">
-              {t("liquidity.asset.details.total")}
+        <div sx={{ flex: "column", gap: 10 }}>
+          <Text fs={13} color="basic400">
+            {t("liquidity.stablepool.asset.details.total")}
+          </Text>
+          <div
+            sx={{
+              flex: "column",
+              align: "start",
+              justify: "center",
+              gap: 4,
+            }}
+          >
+            <Text lh={22} color="white" fs={18}>
+              <DisplayValue value={total.value} />
+            </Text>
+            <Text fs={13} color="basic500">
+              {t("liquidity.asset.details.percent", {
+                value: percentInOmnipool,
+              })}
             </Text>
           </div>
-          <DollarAssetValue
-            value={totalDisplay}
-            wrapper={(children) => (
-              <Text fs={18} lh={22} color="white" tAlign={["right", "left"]}>
-                {children}
-              </Text>
-            )}
-          >
-            <DisplayValue value={totalDisplay} />
-          </DollarAssetValue>
         </div>
-
         <div sx={{ flex: "column", gap: 10, width: ["auto", 118] }}>
           <div sx={{ flex: "row", align: "center", gap: 6 }}>
             <Text fs={13} color="basic400">
-              {t("liquidity.stablepool.asset.details.total")}
+              {t("liquidity.asset.details.24hours")}
             </Text>
+            <InfoTooltip text={t("liquidity.asset.details.24hours.tooltip")}>
+              <SInfoIcon />
+            </InfoTooltip>
           </div>
-          <DollarAssetValue
-            value={total}
-            wrapper={(children) => (
-              <Text fs={18} lh={22} color="white" tAlign={["right", "left"]}>
-                {children}
-              </Text>
-            )}
-          >
-            <DisplayValue value={total} />
-          </DollarAssetValue>
+          {isLoading ? (
+            <Skeleton />
+          ) : (
+            <DollarAssetValue
+              value={data?.volume ?? BN_NAN}
+              wrapper={(children) => (
+                <Text fs={18} lh={22} color="white" tAlign={["right", "left"]}>
+                  {children}
+                </Text>
+              )}
+            >
+              <DisplayValue value={data?.volume} />
```

Comment:
use translation `value.dollar` instead of DisplayValue component. volume is returned as a dollar unit and can't be converted to another asset